latex/headers/common-byval.h

```diff
@@ -5,15 +5,17 @@ class T {
   ...
 
  public:
-  T(const T &rhs) = default;
+  // Rule of Zero: the following five special member functions should not be explicitly declared
```

Comment:
If they are to be uncommented, then we are not following the rule of zero, because it is never correct to have all five of those special member functions inlined, public and defaulted.

Reply:
SYCL working group: We are happy to leave the declarations commented out, though we should update the comment slightly to reflect both the rule of zero and the rule of five, i.e. they either have to all be declared or none should be declared.

Reply:
The comments have been updated.